packages/strapi-plugin-i18n/admin/src/components/CMEditViewLocalePicker/Wrapper.js

```diff
@@ -1,6 +1,7 @@
 import styled from 'styled-components';
 
 const Wrapper = styled.div`
+  padding-bottom: ${({ canCopy }) => (canCopy ? '19px' : '29px')};
```

Comment:
one day, we'll see `padding: 16.66666px` 😆

Reply:
Also, not sure "canCopy" is a great way since it glues the component to the place it's called. Maybe passing a "size" would sound better?